tf-managed/modules/daily-snapshot/variable.tf

```diff
@@ -109,6 +109,13 @@ variable "environment" {
   type        = string
 }
 
+
+variable "snapshot_type" {
```

Comment:
It'd be more extensible (and readable) if we accept a set of networks. On top of that, if we restrict possible values via comment, we can as well restrict them via validations.

Reply:
True, done.

Reply:
i've added the validations, but i'm abit confused about the `accept a set of network`.

Reply:
ping @LesnyRumcajs

Reply:
```
 type    = set(string)
  default = ["mainnet", "calibnet"]
```
Then, in the script, you would iterate over those values and generate proper cron jobs.

Reply:
This approach is more flexible than the one with `both`, as it allows to possibly add more values in the future. Say, we add `pineconet`. Now the allowed values would be `mainnet, calibnet, pineconenet, both`? Seems weird. We'd change `both` to `all` but that would be a breaking change. We should design, if possible, for future changes.

Reply:
sure, done @LesnyRumcajs

Reply:
`all` should not be a network name. The variable itself should be named `snapshot_networks` and, for the moment, it should accept either `mainnet`, `calibnet` or `mainnet,calibnet`. By default `mainnet,calibnet`

Reply:
@LesnyRumcajs done